### PR TITLE
feat: 🎸 toolbar migration

### PR DESCRIPTION
**Pull request by xiaoyu0722**
Differences to `SAPFiori`
No button types, ~~(FUIButton, icon, UIButton, TertiaryButton)~~
No primary secondary or 3rd button
All these are actually same thing in `SwiftUI`: `View`
Logic for items layout arrangement is more simple and clear.

### CODE REVIEW

**Comment by dyongxu** [comment]
@xiaoyu0722 When all the toolbar buttons exceed the width of screen, we still need to keep at least one button, then move others into the overflow button.

**Comment by xiaoyu0722** [comment]
> @xiaoyu0722 When all the toolbar buttons exceed the width of screen, we still need to keep at least one button, then move others into the overflow button.

@dyongxu  Fixed.

File: Apps/Examples/Examples/FioriSwiftUICore/ToolbarExample/ToolbarExample.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            
+            Picker("Button Type", selection: $buttonType) {
+                Text("Fiori Button").tag(ItemStyle.fiori)
+                Text("Icon").tag(ItemStyle.icon)
+                Text("SiwftUI Button").tag(ItemStyle.button)
+            }
+            
+            Picker("Helper Text", selection: $helperText) {
+                Text("None").tag("")
+                Text("Shrot").tag("Helper Text")
```
Comment (dyongxu) [inline Apps/Examples/Examples/FioriSwiftUICore/ToolbarExample/ToolbarExample.swift:48]: Typo: Shrot

Comment (xiaoyu0722) [reply Apps/Examples/Examples/FioriSwiftUICore/ToolbarExample/ToolbarExample.swift]: My bad, fixed

**Comment by xiaoyu0722** [comment]
This branch is protected, I can't update anymore, move to https://github.com/SAP/cloud-sdk-ios-fiori/pull/634

**Pull request closed without merging**